.github/workflows/docker-build-and-push-image.yml

```diff
@@ -20,9 +20,6 @@ on:
     secrets:
       dockerhub_token:
         required: true
-  push:
-    tags:
-      - "v*.*.*"
```

Comment:
Extra: this trigger was wrong, since we only want to trigger this by workflow calls
example error https://github.com/yeagerai/genlayer-simulator/actions/runs/10490636510/job/29057821846#step:3:7